mycpp/TEST.sh

```diff
@@ -240,11 +240,6 @@ compare-examples() {
 }
 
 test-sweep-asan-leaks() {
-  ninja _bin/cxx-sweepasan/mycpp/examples/fib_iter.mycpp
-  ASAN_OPTIONS='' _bin/cxx-sweepasan/mycpp/examples/fib_iter.mycpp
-
-  # CURRENTLY 9 of 22 fail
-  # This does NOT check if all pass
   ASAN_OPTIONS='' examples-variant '' sweepasan
```

Comment:
How many of these pass with ASAN now?  All?  

I think this comment is saying that it doesn't exit 1 if there are failures, which we should fix

So we shouldn't delete the comment yet

Reply:
There's still one failing test, but it's got nothing to do with the GC.  There's some pretty questionable code in the pretty-printers which leak and is not super easy to fix.